python/lumifit/simulation.py

```diff
@@ -172,7 +172,7 @@ def create_simulation_and_reconstruction_job(
     dirname_full = dirname + "/" + dirname_filter_suffix
     pathname_full = lmdfit_data_dir + "/" + dirname_full
     # TODO: this path depends on KOALA/LUMI, so change it accroding to scenario type!
-    macropath_full = os.environ["VMCWORKDIR"] + "/macro/detectors/lmd"
```

Comment:
we shouldn't hardcode this at all, there needs to be a reference to the config here